examples/proto/examplepb/a_bit_of_everything.swagger.json

```diff
@@ -421,6 +423,13 @@
               "TRUE"
             ],
             "default": "FALSE"
+          },
+          {
+            "name": "int64_override_type",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "format": "int64"
```

Comment:
Is this format value well defined?

Reply:
It appears to be according to this: https://swagger.io/docs/specification/data-models/data-types/#numbers

From what I have read, the `type` field must consist of one of the defined types (for which there are enums) and the format field is somewhat arbitrary. 
Ref: https://swagger.io/docs/specification/data-models/data-types/#string
At least for strings it says that `format` is an open value. Under the `integer` it says that format is a hint for whatever program is using it.